webapp/i18n/i18n.properties

```diff
@@ -0,0 +1,13 @@
+ListReportTitle=Products
+ProductText=Product
+NameText=Name
```

Comment:
Name it ProductName because you are going more names in the app.  Product Name, Supplier Name, Category Name and so on.

My advice is to use the following pattern:

Data entity -> Its property -> Control
Product           Name                Text

So the result is
ProductNameText

Reply:
Please review all i18n keys